ffmpeg/_ffmpeg.py

```diff
@@ -24,6 +24,27 @@ def input(filename, **kwargs):
     return InputNode(input.__name__, kwargs=kwargs).stream()
 
 
+
+def source_multi_output(filter_name, *args, **kwargs):
```

Comment:
Are there any multi-output sources that you know of? If not maybe we should drop `source_multi_output`.

Reply:
In any case if there is it'd be good to add an example in the docstring of a source that has multiple outputs.

Reply:
Actually I have no idea, I just added it for completeness. 👍 for docstring and tests.
Later this week or next week.